packages/react/src/date-picker/Calendar.tsx

```diff
@@ -387,34 +392,54 @@ const calendarComponents: CustomComponents = {
 		};
 
 		const interactiveProps = {
-			'aria-current': activeModifiers.today ? 'date' : undefined,
+			'aria-current': modifiers.today ? 'date' : undefined,
 			// Improve the aria labels of each button.
 			// Selected and dates within range are manually announced.
 			'aria-label': `${
-				activeModifiers.selected && !activeModifiers.range_middle
-					? 'Selected. '
-					: ''
-			}${formatHumanReadableDate(props.date)}${
-				activeModifiers.range_middle ? '. Between selected dates' : ''
+				modifiers.selected && !modifiers.range_middle ? 'Selected. ' : ''
+			}${formatHumanReadableDate(day.date)}${
+				modifiers.range_middle ? '. Between selected dates' : ''
 			}`,
 			'aria-selected':
 				// React Day Picker incorrectly marks ranges as selected
-				activeModifiers.range_middle ? undefined : activeModifiers.selected,
+				modifiers.range_middle ? undefined : modifiers.selected,
 			onClick,
-			...restButtonProps,
 		};
 
+		// react-day-picker v9 the class names `range_start` and `range_end` are only given once both start and end dates are selected
```

Comment:
This sentence doesn't read correctly. Is it "in react-day-picker v9, the class names…"?